Sources/FoundationEssentials/FileManager/FileManager+Utilities.swift

```diff
@@ -273,15 +273,23 @@ extension _FileManagerImpl {
 
 #if !os(Windows)
     static func _userAccountNameToNumber(_ name: String) -> uid_t? {
+        #if os(WASI) // WASI doesn't have getpwnam
```

Comment:
We don't compile these at all for Windows since they don't make sense on that platform, should we just do the same here (i.e. `#if !os(Windows) && !os(WASI)` above) so that we don't accidentally try to call these on WASI?

Reply:
Oh, we don't use these methods other than in `setAttributes`, so we can exclude them at all, thanks!